filebeat/module/mariadb/slowlog/_meta/fields.yml

```diff
@@ -0,0 +1,73 @@
+- name: slowlog
+  type: group
+  release: experimental
+  description: >
+    Contains fields from the MySQL slow logs.
+  fields:
+    - name: user
+      description: >
+        The MySQL user that created the query.
+    - name: current_user
+      description: >
+        The MySQL user that created the query.
+    - name: host
+      description: >
+        The host from where the user that created the query logged in.
+    - name: client_host
+      description: >
+        The host from where the user that created the query logged in.
+    - name: ip
+      description: >
+        The IP address from where the user that created the query logged in.
+    - name: client_ip
+      description: >
+        The IP address from where the user that created the query logged in.
+    - name: query_time.sec
```

Comment:
I thought that this type of fields like `sec` must be in a sub-group and not as part of the field name with a `.` (dot)

Reply:
See comment here https://github.com/elastic/beats/pull/9414#discussion_r241356110